jib-gradle-plugin/README.md

```diff
@@ -219,7 +219,7 @@ Property | Type | Default | Description
 
 Property | Type | Default | Description
 --- | --- | --- | ---
-`image` | `String` | *Required* | The image reference for the target image. This can also be specified via the `--image` command line option.
+`image` | `String` | *Required* | The image reference for the target image. This can also be specified via the `--image` command line option. If the tag is not present here :latest is implied.
```

Comment:
```suggestion
`image` | `String` | *Required* | The image reference for the target image. This can also be specified via the `--image` command line option. If the tag is not present here `:latest` is implied.
```